src/main/java/org/launchcode/VetConnect/models/AbstractEntity.java

```diff
@@ -1,19 +1,18 @@
 package org.launchcode.VetConnect.models;
 
 
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.persistence.MappedSuperclass;
+import javax.persistence.*;
+import javax.persistence.criteria.CriteriaBuilder;
 import java.util.Objects;
 
 @MappedSuperclass
 public abstract class AbstractEntity {
 
     @Id
-    @GeneratedValue
-  private int id;
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
Good idea to change to Long.  If you want to make this change, In all of the Repositories, there's currently an Integer type that represents the type of the id that I think might need to be changed to Long.  For example:  public interface ClinicRepository extends CrudRepository<Clinic, **Integer**>